shared/transforms/xccdf-addremediations.xslt

```diff
@@ -71,7 +73,9 @@
     - NIST-800-53-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_nist800_171">
     - NIST-800-171-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_pci_dss">
     - PCI-DSS-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_cjis">
-    - CJIS-<xsl:value-of select="."/></xsl:for-each></xsl:variable>
+    - CJIS-<xsl:value-of select="."/></xsl:for-each><xsl:if test="$ref_disa_ossrg"><xsl:for-each select="$ref_disa_ossrg">
```

Comment:
What is the if for? why is it needed for OSSRG and not for other references? shouldn't for-reach over empty node set just not iterate at all?

Reply:
It looks like the `<if>` isn't necessary.